src/emscripten-optimizer/colors.h

```diff
@@ -17,6 +17,7 @@
 #ifndef wasm_color_h
 #define wasm_color_h
 
+#ifndef WIN32
 #include <unistd.h>
```

Comment:
Ah right, can you just wrap the use() stuff in `#if defined(__linux__) || defined(__apple__)` like below? Actually, probably just factor out that #if into its own macro.